bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/WebSocketConnection.java

```diff
@@ -64,6 +68,9 @@
     Listener listener;
     boolean closed;
     IWebSocketCommandHandler webSocketCommandHandler;
+    private long messageCounter = 0;
```

Comment:
Where is this counter and the one below it being used? I see them being incremented, but i don't see where anything is done with that count

Reply:
Good point. I originally thought that unfinished requests lead to the shutdown faults and logged the sent and received messages on close. I forgot to remove the counters later.